products.js

```diff
@@ -37,8 +37,9 @@ function formatPrice(priceString){
  * Remove the query params from a url by removing the '?' and the text that comes after it
  */
 function withoutQueryParams(url){
-  if (url.includes('?')){
-    return url.substr(0,url.indexOf('?'));
+  var index = url.includes('?');
```

Comment:
Read your diffs and run the tests before committing. Always. :-) (If you don't have tests, test it manually. Then the pain will drive you to write tests.)